saga/seata-saga-spring/src/main/java/org/apache/seata/saga/engine/store/db/DbAndReportTcStateLogStore.java

```diff
@@ -729,11 +729,27 @@ public List<StateInstance> queryStateInstanceListByMachineInstanceId(String stat
     }
 
     @Override
-    public void clearUp() {
+    public void clearUp(ProcessContext context) {
         RootContext.unbind();
         RootContext.unbindBranchType();
         if (sagaTransactionalTemplate != null) {
-            sagaTransactionalTemplate.cleanUp();
+            GlobalTransaction globalTransaction;
+            StateMachineInstance machineInstance =  (StateMachineInstance) context.getVariable(DomainConstants.VAR_NAME_STATEMACHINE_INST);
+            try {
+                globalTransaction = getGlobalTransaction(machineInstance, context);
+                if (globalTransaction == null) {
+
+                    throw new EngineExecutionException("Global transaction is not exists",
+                            FrameworkErrorCode.ObjectNotExists);
+                }
+                sagaTransactionalTemplate.cleanUp(globalTransaction);
+            } catch (ExecutionException e) {
+                LOGGER.error("Report transaction finish to server error: {}, StateMachine: {}, XID: {}, Reason: {}",
+                        e.getCode(), machineInstance.getStateMachine().getName(), machineInstance.getId(), e.getMessage(), e);
+            } catch (TransactionException e) {
+                LOGGER.error("Report transaction finish to server error: {}, StateMachine: {}, XID: {}, Reason: {}",
+                        e.getCode(), machineInstance.getStateMachine().getName(), machineInstance.getId(), e.getMessage(), e);
+            }
```

Comment:
First, there is no need to throw `ExecutionException` and re-catch it, and I prefer not to log error for "global transaction not found".
Besides, it is not possible for these lines to throw `TransactionException`. Remove it in case to confuse others.

In one word, it would be nice to remove these loggings.

Reply:
Sry, I miss one line above. Dont care about these comments.